lib/kitchen/provisioner/ansible_push.rb

```diff
@@ -39,10 +40,108 @@ class AnsiblePush < Base
       # For tests disable if not needed
       default_config :chef_bootstrap_url, "https://www.getchef.com/chef/install.sh"
 
+      attr_reader :conf
```

Comment:
but isn't it true that in Ruby when calling a no-argument method you don't have to specify parentheses? For example:
```
irb(main):004:0> def x
irb(main):005:1> { :a => 1, :b => 2 }
irb(main):006:1> end
=> :x
irb(main):007:0> x[:a]
=> 1
```
To my understanding  ```attr_reader conf```` and ```attr_reader :machine_name`` is redundant